eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM spotify/kafka
+FROM buildpack-deps:xenial-curl
 
 #
 # Install apt packages needed by backbeat and buildbot_worker
 #
+ENV LANG C.UTF-8
 COPY ./backbeat_packages.list ./buildbot_worker_packages.list /tmp/
 
-# Workaround since Jessie release has been archived recently
-# (https://lists.debian.org/debian-devel-announce/2019/03/msg00006.html)
-RUN rm -f /etc/apt/sources.list.d/jessie-backports.list
-
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+RUN curl -sS http://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
```

Comment:
I believe this is installing yarn, but I don't see it being used in the changes.

Reply:
We are installing it as we need to shift to yarn in the future. That would be next.
This is just node upgrade so as we were changing dockerfile added that too.
moving to yarn changes was intentionally not added in the pull request. Only the installing part.